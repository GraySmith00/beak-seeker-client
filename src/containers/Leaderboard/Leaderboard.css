.leaderboard {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin-bottom: 2rem;
  height: 100%;
  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("./birdBackground6.jpg");
  background-size: cover;
  background-position: top center; }
  .leaderboard .main-content {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    justify-content: center;
    padding-top: 4.5rem;
    width: 80%; }
  .leaderboard .leader-table {
    width: 100%; }
  .leaderboard .leader-row {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin: 0.5rem auto;
    padding: 0.5rem 1rem;
    width: 100%;
    background: rgba(0, 0, 0, 0.75);
    border-radius: 5px;
    color: #fff;
    -webkit-box-shadow: 10px 10px 26px -9px rgba(0, 0, 0, 0.43);
    -moz-box-shadow: 10px 10px 26px -9px rgba(0, 0, 0, 0.43);
    box-shadow: 10px 10px 26px -9px rgba(0, 0, 0, 0.43); }
  .leaderboard .checkbox-input {
    transform: scale(1.5);
    border: none; }
  .leaderboard .twitter-button {
    padding: 1rem;
    text-decoration: none;
    margin-top: 1.5rem;
    margin-bottom: 6rem;
    width: 100%;
    text-align: center;
    border-radius: 5px;
    color: #fff;
    background: #00c6ff;
    /* fallback for old browsers */
    background: -webkit-linear-gradient(to right, #0072ff, #00c6ff);
    /* Chrome 10-25, Safari 5.1-6 */
    background: linear-gradient(to right, #0072ff, #00c6ff);
    /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
    -webkit-box-shadow: 10px 10px 43px -8px rgba(0, 0, 0, 0.43);
    -moz-box-shadow: 10px 10px 43px -8px rgba(0, 0, 0, 0.43);
    box-shadow: 10px 10px 43px -8px rgba(0, 0, 0, 0.43); }
