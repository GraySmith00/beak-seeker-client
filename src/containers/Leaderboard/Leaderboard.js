import React, { Component } from 'react';

class Leaderboard extends Component {
  render() {
    return (
      <div>
        <h1>Leaderboard</h1>
      </div>
    );
  }
}

export default Leaderboard;
