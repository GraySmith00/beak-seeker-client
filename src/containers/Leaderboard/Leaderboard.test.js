import React from 'react';
import { shallow } from 'enzyme';

import Leaderboard from './Leaderboard';

describe('Leaderboard component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Leaderboard />);
  });

  it('it should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
