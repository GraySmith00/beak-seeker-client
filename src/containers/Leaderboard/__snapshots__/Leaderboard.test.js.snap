// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Leaderboard component it should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Leaderboard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <withRouter()
          currentPage="Leaderboard"
/>,
        <div
          className="main-content"
>
          <div
                    className="leader-row"
          >
                    <span>
                              Place
                    </span>
                    <span>
                              Name
                    </span>
                    <span>
                              Sightings
                    </span>
          </div>
</div>,
      ],
      "className": "leaderboard",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "currentPage": "Leaderboard",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="leader-row"
>
              <span>
                            Place
              </span>
              <span>
                            Name
              </span>
              <span>
                            Sightings
              </span>
</div>,
            undefined,
          ],
          "className": "main-content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <span>
                  Place
</span>,
                <span>
                  Name
</span>,
                <span>
                  Sightings
</span>,
              ],
              "className": "leader-row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Place",
                },
                "ref": null,
                "rendered": "Place",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Name",
                },
                "ref": null,
                "rendered": "Name",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sightings",
                },
                "ref": null,
                "rendered": "Sightings",
                "type": "span",
              },
            ],
            "type": "div",
          },
          undefined,
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <withRouter()
            currentPage="Leaderboard"
/>,
          <div
            className="main-content"
>
            <div
                        className="leader-row"
            >
                        <span>
                                    Place
                        </span>
                        <span>
                                    Name
                        </span>
                        <span>
                                    Sightings
                        </span>
            </div>
</div>,
        ],
        "className": "leaderboard",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "currentPage": "Leaderboard",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="leader-row"
>
                <span>
                                Place
                </span>
                <span>
                                Name
                </span>
                <span>
                                Sightings
                </span>
</div>,
              undefined,
            ],
            "className": "main-content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <span>
                    Place
</span>,
                  <span>
                    Name
</span>,
                  <span>
                    Sightings
</span>,
                ],
                "className": "leader-row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Place",
                  },
                  "ref": null,
                  "rendered": "Place",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Name",
                  },
                  "ref": null,
                  "rendered": "Name",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sightings",
                  },
                  "ref": null,
                  "rendered": "Sightings",
                  "type": "span",
                },
              ],
              "type": "div",
            },
            undefined,
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
